arenadata/Dockerfile.ubuntu

```diff
@@ -29,27 +29,42 @@ RUN set -eux; \
     pip2 install allure-behave==2.4.0; \
     pip2 cache purge
 
+RUN wget https://www.python.org/ftp/python/3.9.19/Python-3.9.19.tgz -O - | tar zxf - && \
+    cd Python-3.9.19 && \
+    ./configure --prefix=/usr/local --enable-shared --enable-optimizations && \
+    make -j6 && \
+    make install && \    
+    echo "/usr/local/lib" >>/etc/ld.so.conf.d/gpdb.conf && \
+	ldconfig && \
+    cd .. && \
+    rm -rf Python-3.9.19.tgz Python-3.9.19
```

Comment:
rewrite in moder design with `set -eux` and `;` instead `&&` as above
use `pushd` and `popd` instead `cd`

Reply:
Any references why do you consider using of set -eux "moder" and better ?

Reply:
It is used by [last official PostgreSQL](https://github.com/docker-library/postgres/blob/3a94d965ecbe08f4b1b255d3ed9ccae671a7a984/16/bookworm/Dockerfile#L32-L50)

Reply:
Are there any advantages of it or we shall use just because someone else did it?

Reply:
we can use if else simplier